tests/unit/amo/pages/TestCollection.js

```diff
@@ -1091,4 +1091,30 @@ describe(__filename, () => {
       expect(extractId(props)).toEqual('123/collection-bar/124');
     });
   });
+
+  describe('Tests for CollectionSort', () => {
+    it.each([true, false])(
+      `calls history.push with expected pathname and query when a sort is selected and editing is %s`,
+      (editing) => {
+        const location = `${defaultLocation}${editing ? 'edit/' : ''}`;
```

Comment:
this seems to be manually replicating the logic with `editing` in `getLocation`?

Reply:
Good catch. I refactored the test suite a number of times and didn't update this one. I'll do that.